openprescribing/templates/measure_for_one_entity.html

```diff
@@ -12,20 +12,22 @@
 {% block content %}
   <h1>
     {{ measure.name }}
-    <br><small>{{ entity.name }}</small>
+    {% if entity %}<br><small>{{ entity.name }}</small>{% endif %}
```

Comment:
Rather than have two `{% if entity %}...{% endif %}` blocks, have one.  You'll duplicate part of the `<h1>` element, but the template might be clearer.